woocommerce/woocommerce-ios
# [Order Creation] Add Collect Payment button to Order Form

Closes: #11466 
Merge after: #11482 


## Description
We are improving the Order Creation form on tablets. The components used will be shared between iPhone and iPad, and one of the changes we identified to support this is enabling payment from the Order Creation form, without first having to tap `Create`.

This adds a `Collect Payment` button to the form; in #11467 it will be moved to a floating/expandable panel to ensure it's always easy to get hold of. For the sake of this change, it's added in line with the order totals, which will also be shown in the expandable panel.

## Testing instructions


1. Launch the app
2. Navigate to the `Orders` tab
3. Tap `+`
4. Observe that there's a (disabled) Collect Payment button
5. Add a product to the order
6. Observe that the Collect Payment button is enabled
7. Tap Collect Payment
8. Observe that the Payment Methods screen is shown modally, and the order details are pushed underneath (more obvious on iPad)
9. Select a card reader or TTP payment method
10. Observe that you can pay for the order, and when you're done, you get taken to the Order Details screen

## Screenshots


https://github.com/woocommerce/woocommerce-ios/assets/2472348/b01d6372-7ac6-4965-85a4-1f4bd56f1e4e



---
